app/controllers/posts_controller.rb

```diff
@@ -2,11 +2,34 @@ class PostsController < ApplicationController
   def index
     @user = User.find(params[:user_id])
     @posts = @user.posts.includes(:comments)
-    # puts @posts
   end
 
   def show
     @user = User.find(params[:user_id])
     @post = Post.find(params[:id])
   end
+
+  def new
+    @post = Post.new
+    @user = User.find(params[:user_id])
+    @current_user = current_user
```

Comment:
[OPTIONAL] This is unnecessary for the new action since its purpose is just to create a form for creating new posts.